app/components/pull-request-page-changeset-wrapper.js

```diff
@@ -0,0 +1,15 @@
+import Ember from "ember"
```

Comment:
Since this file isn't doing anything anymore, I'd just remove it and call `pull-request-page` from the parent component.